protocol/chat.go

```diff
@@ -150,7 +150,16 @@ func (c *Chat) MembersAsPublicKeys() ([]*ecdsa.PublicKey, error) {
 	return stringSliceToPublicKeys(publicKeys, true)
 }
 
-func (c *Chat) updateChatFromProtocolGroup(g *v1protocol.Group) {
+func (c *Chat) updateChatFromProtocolGroup(myID string, g *v1protocol.Group) {
+
+	// Check if we were already in the chat
+	var hadJoined bool
+	for _, member := range c.Members {
+		if myID == member.ID {
+			hadJoined = true
```

Comment:
Wouldn't it be cleaner to have a `func (c *Chat) HasMember(memberId string) bool` method?

Reply:
I thought this, but it also introduces an extra iteration over the `c.Members`

Reply:
Ah sorry ignore me. I'm looking at the wrong place in the code 🤦

Reply:
`func (c *Chat) HasMember(memberId string) bool` would be an improvement.